### DISCUSSION TITLE
How can I change the default html template for the image plugin when image is uploaded on the editor?

### DISCUSSION THREAD

**Discussion by yan-g-s-machado** [JiHong88/SunEditor#1293] [Q&A]
Hi, 

Thanks for this  WYSIWYG web editor! It's really good, but I gotta a little problem.

When a user uploads some image, I need the image plugin do not includes `<figure></figure>` tags and its style on the editor. I just want the image tag with its source attribute, for example.  

How can I do it?

Look at the examples below:

**This is the default:**

```
<div class="se-component se-image-container __se__float-none">
  <figure style="width: 3cm;"> <!-- I need to remove this and its style -->
    <img src="some-base64-data-here" data-file-size="0" origin-size="300,300" data-origin="3cm,3cm" data-size="3cm,3cm" style="width: 3cm; height: 3cm" data-index="5">
  </figure> <!-- I need to remove this and its style -->
</div>
```

**This is what I need:**

```
<div class="se-component se-image-container __se__float-none">
    <img src="some-base64-data-here" data-file-size="0" origin-size="300,300" data-origin="3cm,3cm" data-size="3cm,3cm" style="width: 3cm; height: 3cm" data-index="5"
</div>
```

**Comment by JiHong88** [comment]
The figure tag is related to image selection and editing, so it is difficult to allow other structures together.
